libs9s/s9srpcclient.cpp

```diff
@@ -6615,8 +6615,12 @@ S9sRpcClient::removeNode()
     title.sprintf("Remove '%s' from the Cluster", STR(hostName));
 
     // The job_data...
-    jobData["nodes"]      = nodesField(hosts);
-    jobData["enable_uninstall"] = options->uninstall();
+    jobData["nodes"] = nodesField(hosts);
+
+    bool const doUninstall = options->uninstall();
+    jobData["enable_uninstall"] = doUninstall;
+    if (!doUninstall)
+        jobData["unregister_only"] = options->unregisterOnly();
```

Comment:
Lets not add such smarties.

If we do not uninstall, we still do remove the node from the cluster.

Unregister only term was used when we unregister a cluster from CC, but do not change its configuration and keep it running.

Reply:
Sorry, I misunderstood this code part.

Yet for being protective, to avoid unwanted harm it would be better to start by checking for unregister only, like so:
```
if (options->unregisterOnly())
    jobData["unregister_only"] = true;
else {
    jobData["enable_uninstall"] = options->uninstall();
}
```

Reply:
We can not have this, this is an invalid use case:
```
s9s  cluster \
    --remove-node \
    --uninstall \
    --unregister-only \
    ...
```
Also as much as this command we could just have to alternate (still invalid):
```
s9s  cluster \
    --remove-node \
    --unregister-only \
    --uninstall \
    ...
```
We don't have to expect any of these.
The difference is which of these will be default if we fail to reject it. And if we fail to reject this invalid case, it is more safe not to delete.